tide_core.install

```diff
@@ -1548,3 +1548,36 @@ function tide_core_update_8064() {
     }
   }
 }
+
+/**
+ * Add contact field to user account(business name, phone, notes).
+ */
+function tide_core_update_8065() {
+  module_load_include('inc', 'tide_core', 'includes/helpers');
+  $config_location = [\Drupal::service('extension.list.module')->getPath('tide_core') . '/config/install'];
+  $configs = [
+    'field.storage.user.field_business_contact_number' => 'field_storage_config',
+    'field.storage.user.field_business_name' => 'field_storage_config',
+    'field.storage.user.field_notes' => 'field_storage_config',
+  ];
+  foreach ($configs as $config => $type) {
+    $config_read = _tide_read_config($config, $config_location, TRUE);
+    $storage = \Drupal::entityTypeManager()->getStorage($type);
+    $id = $storage->getIDFromConfigName($config, $storage->getEntityType()->getConfigPrefix());
+    if ($storage->load($id) == NULL) {
+      $config_entity = $storage->createFromStorageRecord($config_read);
+      $config_entity->save();
+    }
+  }
+}
+
+/**
+ * Update user form display and authenticated user permission.
+ */
+function tide_core_update_8066() {
+  module_load_include('inc', 'tide_core', 'includes/helpers');
+  $config_location = [\Drupal::service('extension.list.module')->getPath('tide_core') . '/config/install'];
+  _tide_import_single_config('user.role.authenticated', $config_location);
+  _tide_import_single_config('user.role.site_admin', $config_location);
+  _tide_import_single_config('core.entity_form_display.user.user.default', $config_location);
+}
```

Comment:
Please do not use this `_tide_import_single_config` anymore, I will add a tag in this function to say deprecated
Also you can create the `entity_form_display` and `entity view display` using the `\Drupal::entityTypeManager()` Please check the following example - https://github.com/dpc-sdp/tide_landing_page/blob/develop/tide_landing_page.install#L1328